pom.xml

```diff
@@ -925,6 +932,42 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>com.diffplug.spotless</groupId>
+        <artifactId>spotless-maven-plugin</artifactId>
+        <version>${spotless.version}</version>
+        <configuration>
+          <java>
+            <endWithNewline />
+            <importOrder />
```

Comment:
@timja this import order might mess with how we have configured in checkstyle, but we could simply adapt to the order so that static was on top.

https://github.com/jenkinsci/configuration-as-code-plugin/blob/1bed6f13340e0d7bdd2c578d9ead394eedd39555/.mvn/checkstyle.xml#L12-L19

Reply:
heh low level of caring what the actual order is as long as it's defined and enforced in code :).

happy to change when we update the parent pom